OpenRA.Game/Widgets/Widget.cs

```diff
@@ -279,21 +279,22 @@ public void PostInit(WidgetArgs args)
 
 		public virtual Rectangle EventBounds { get { return RenderBounds; } }
 
-		public virtual Rectangle GetEventBounds()
+		public virtual bool CheckEventBounds(int2 location)
```

Comment:
`CheckEventBounds` is ambiguous. Can we call this something like `EventBoundsContains`?